cypress/integration/brewery/DiscardAndContinue.spec.js

```diff
@@ -0,0 +1,132 @@
+import { ScenarioManager, Scenarios, ScenarioParameters, Login, Workspaces } from '../../commons/actions';
+import { BreweryParameters } from '../../commons/actions/brewery';
+import { stub } from '../../commons/services/stubbing';
+import utils from '../../commons/TestUtils';
+import { routeUtils as route } from '../../commons/utils';
+import { EXTENDED_WORKSPACES_LIST } from '../../fixtures/stubbing/default';
+
+describe('Discard and continue inside same scenario', () => {
```

Comment:
Did you mean "inside the same **workspace**"?

Reply:
done